buddyboss/buddyboss-platform
# Firefox comment/reply anchoring

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2011 

### How to test the changes in this Pull Request:

1. Go to Newsfeed using Firefox
2. Click on any post comment link
3. See result

### Proof Screenshots or Video

https://drive.google.com/file/d/1q_lAIYYZd8QUhkQEH7sawzHgG0KSaljm/view

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
